app/user_group.go

```diff
@@ -222,6 +222,149 @@ func (c *UserGroupClient) setAccess(ctx *cli.Context, groupID string,
+	aRole := accountRoleToAccess(accountRole)
+	if aRole == nil {
+		return fmt.Errorf("Invalid account role: %s", accountRole)
+	}
+
+	spec := &identity.UserGroupSpec{
+		DisplayName: displayName,
+		Access: &identity.Access{
+			AccountAccess: aRole,
+		},
+		GroupType: &identity.UserGroupSpec_CloudGroup{
+			CloudGroup: &identity.CloudGroupSpec{},
+		},
+	}
+
+	resp, err := c.client.CreateUserGroup(c.ctx, &cloudsvc.CreateUserGroupRequest{
+		Spec: spec,
+	})
+	if err != nil {
+		return err
+	}
+
+	return PrintProto(resp.GetAsyncOperation())
+}
+
+// getUserIDByEmail gets a user ID by email using the auth service API
+func (c *UserGroupClient) getUserIDByEmail(email string) (string, error) {
+	res, err := c.authClient.GetUser(c.ctx, &authservice.GetUserRequest{
+		UserEmail: email,
+	})
+	if err != nil {
+		return "", fmt.Errorf("unable to get user: %w", err)
+	}
+	if res.User == nil || res.User.Id == "" {
+		return "", fmt.Errorf("user not found for email: %s", email)
+	}
+	return res.User.Id, nil
+}
+
+// addUsersToGroup adds users to a group using the cloud services API
+func (c *UserGroupClient) addUsersToGroup(ctx *cli.Context, groupID string, emails []string) error {
+	for _, email := range emails {
+		userID, err := c.getUserIDByEmail(email)
+		if err != nil {
+			return fmt.Errorf("unable to add user %s to group: %w", email, err)
+		}
+
+		req := &cloudsvc.AddUserGroupMemberRequest{
+			GroupId: groupID,
+			MemberId: &identity.UserGroupMemberId{
+				MemberType: &identity.UserGroupMemberId_UserId{
+					UserId: userID,
+				},
+			},
+			AsyncOperationId: ctx.String(RequestIDFlagName),
+		}
+
+		resp, err := c.client.AddUserGroupMember(c.ctx, req)
+		if err != nil {
+			return fmt.Errorf("unable to add user %s to group: %w", email, err)
+		}
+
+		if err := PrintProto(resp.GetAsyncOperation()); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// removeUsersFromGroup removes users from a group using the cloud services API
+func (c *UserGroupClient) removeUsersFromGroup(ctx *cli.Context, groupID string, emails []string) error {
+	for _, email := range emails {
+		userID, err := c.getUserIDByEmail(email)
+		if err != nil {
+			return fmt.Errorf("unable to remove user %s from group: %w", email, err)
+		}
+
+		req := &cloudsvc.RemoveUserGroupMemberRequest{
+			GroupId: groupID,
+			MemberId: &identity.UserGroupMemberId{
+				MemberType: &identity.UserGroupMemberId_UserId{
+					UserId: userID,
+				},
+			},
+			AsyncOperationId: ctx.String(RequestIDFlagName),
+		}
+
+		resp, err := c.client.RemoveUserGroupMember(c.ctx, req)
+		if err != nil {
+			return fmt.Errorf("unable to remove user %s from group: %w", email, err)
+		}
+
+		if err := PrintProto(resp.GetAsyncOperation()); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// listGroupMembers lists the members of a group using the cloud services API
+func (c *UserGroupClient) listGroupMembers(
+	pageToken string,
```

Comment:
I see other list commands doing the pagination automatically: https://github.com/temporalio/tcld/blob/ea55673852b7ab54458e30c4b4922b8352f34f6d/app/namespace.go#L310C27-L327

Do we want to do the same here?